x-pack/plugin/sql/sql-proto/src/main/java/org/elasticsearch/xpack/sql/proto/SqlClearCursorRequest.java

```diff
@@ -14,16 +14,22 @@
 public class SqlClearCursorRequest extends AbstractSqlRequest {
 
     private final String cursor;
+    private final boolean binaryCommunication;
 
-    public SqlClearCursorRequest(String cursor, RequestInfo requestInfo) {
+    public SqlClearCursorRequest(String cursor, RequestInfo requestInfo, boolean binaryMode) {
         super(requestInfo);
         this.cursor = cursor;
+        this.binaryCommunication = binaryMode;
     }
 
     public String getCursor() {
         return cursor;
     }
 
+    public boolean isBinaryCommunication() {
```

Comment:
```suggestion
    public boolean binaryCommunication() {
```

to keep in in sync with action's version.